docs/models.yml

```diff
@@ -126,18 +126,17 @@ user:
   is_present_in_meeting_ids:
     type: relation-list
     to: meeting/present_user_ids
-  meeting_id:
-    type: relation
-    to: meeting/temporary_user_ids  # Temporary users
-  guest_meeting_ids:
-    type: relation-list
-    to: meeting/guest_ids  # Guests in meetings
-  committee_as_member_ids:
-    type: relation-list
-    to: committee/member_ids
-  committee_as_manager_ids:
+  committee_ids:
     type: relation-list
-    to: committee/manager_ids
+    to: committee/user_ids
+
+  # committee specific permissions
+  committee_$_management_level:
+    type: template
+    replacement: committee_id
+    description: Hierarchical permission level for the each committee organisation.
+    enum:
+    - manager
```

Comment:
I would suggest to keep the naming scheme of our other permissions and name this `can_manage`

Reply:
But regardless of that, it says `manage` instead of `manager` in the `initial_data`. Shouldn't this be caught by the CI? Can you look into this?

Reply:
structured fields are not implemented by the datavalidator yet.

Reply:
Ah ok. But checking just the value (ignoring every aspect of them being structured) should be easy to implement, right?

Reply:
structured fields checking in general should not be a huge problem. Maybe I find some spare time...

(Currently, every field containing a `$` is just skipped for the sake of simplicity)